app/(pages)/products/product-view/_components/ProductView.tsx

```diff
@@ -0,0 +1,143 @@
+    if (currentImage === 1) {
+      setCurrentImage(displayImages.length);
+    } else {
+      setCurrentImage(currentImage - 1);
+    }
+  }
+
+  const displayImages = [
+    '/sample-product/shirt1.png',
+    '/sample-product/shirt2.png',
+    '/sample-product/shirt3.png',
+    '/sample-product/shirt4.png',
+    '/sample-product/shirt5.png',
+    '/sample-product/shirt6.png',
+  ];
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.main_content}>
+        <div className={styles.item_images}>
+          <div className={styles.current_item}>
+            <Image
+              src={displayImages[currentImage - 1]}
+              alt="image one of shirt"
+              fill={true}
+            />
+          </div>
+          <div className={styles.gallery}>
+            {displayImages.map((image, index) => (
+              <li
+                key={index}
+                className={`${
+                  currentImage === index + 1
+                    ? styles.highlight_gallery
+                    : styles.gallery_image
+                }`}
+              >
+                <Image src={image} alt="image gallery display" fill={true} />
+              </li>
+            ))}
+          </div>
+        </div>
+        <div className={styles.item_info}>
+          <div className={styles.info_group}>
+            <div className={styles.item_status}>
+              <Image
+                src={`${
+                  fakeItem.listed
+                    ? '/icons/check_circle_fill.svg'
+                    : '/icons/close_circle_fill.svg'
+                }`}
+                alt="icon if listed"
+                width={24}
+                height={24}
+              />
+              <p>{fakeItem.listed ? 'Listed' : 'Not Listed'}</p>
+            </div>
+            <p className={styles.item_title}>{fakeItem.name}</p>
+            <h4>${fakeItem.price}</h4>
+          </div>
+          <div className={styles.info_group}>
+            <div className={styles.row_container}>
+              <p className={styles.info_label}>Tags:</p>
+              <div className={styles.tags_container}>
+                {fakeItem.tags.map((tag, index) => (
+                  <p key={index} className={styles.tag}>
+                    {tag}
+                  </p>
+                ))}
+              </div>
+            </div>
+          </div>
+          <div className={styles.info_group}>
+            <div className={styles.row_container}>
+              <p className={styles.info_label}>Description:</p>
+              <p>{fakeItem.description}</p>
+            </div>
+            <div className={styles.row_container}>
+              <p className={styles.info_label}>Details:</p>
+              <p>{fakeItem.details}</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className={styles.slider}>
+        <button onClick={subCurrentImage}>
+          <Image
+            src="/icons/chevron-left.svg"
```

Comment:
For any image that you expect to be pulled locally (icons, static images) you should use static image rendering

https://nextjs.org/docs/app/building-your-application/optimizing/images#local-images

This only applies to the icons on this page tho, the shirt images are expected to come externally so those are fine.

Reply:
Added! (I think)